plentyofpets/lib/components/admin_homepage_buttons

```diff
@@ -0,0 +1,134 @@
+import 'package:flutter/material.dart';
+import 'package:admin_homepage_navigation/add_pet.dart';
+
+class AddPetCardButton extends StatelessWidget {
+  final String buttonText;
+
+  AddPetCardButton({required this.buttonText});
```

Comment:
There is some convention where things that extend StatelessWidget are supposed to include a parameter of type Key and called super() with the Key. You can take a look [here](https://github.com/fraszle/CS467_PetAdoption/blob/main/plentyofpets/lib/components/decorations.dart). VSCode can actually add some of this boilerplate code for you if you look for the warning highlights